JDBC/OAuthSampleApp/src/main/java/OAuthSampleApp.java

```diff
@@ -70,14 +147,128 @@ private static void printResults(ResultSet rs) throws SQLException {
             rowIdx++;
         }
     }
+    
+    private static String getEncodedParamsString(Map<String, String> params) throws UnsupportedEncodingException {
```

Comment:
What is this function doing and why is it needed? For a public sample application, users will be confused by this.

Reply:
This function converts the normal strings to url encoded string. In ADO.Net there is direct function to do that. 
JDBC has thirdparty library which we dont want to use. So We have added the method to do same work using Java library.

Reply:
OK. Just leave a short comment in the code explaining what it does.